### ISSUE TITLE
[Bug Report][3.3.9] Keyboard navigation doesn't work like the Vuetify2

### ISSUE THREAD

**Issue by sametaylak** [vuetifyjs/vuetify#18011]
### Environment
**Vuetify Version:** 3.3.9
**Last working version:** 2.6.12
**Vue Version:** 3.3.4
**Browsers:** Chrome 115.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
- Click the VMenu activator
- Press ArrowDown in keyboard

### Expected Behavior
It should focus list items

### Actual Behavior
It doesn't focus list items

### Reproduction Link
[https://codesandbox.io/s/amazing-engelbart-w2tmkv](https://codesandbox.io/s/amazing-engelbart-w2tmkv)

### Other comments
You can find Vuetify2 version in here: https://codesandbox.io/s/cranky-diffie-hwkcl8
Vuetify3 verions in here: https://codesandbox.io/s/amazing-engelbart-w2tmkv

If you see cannot use import error, please try to refresh preview view

**Comment by KaelWD** [comment]
It works if you put everything in one list.

**Comment by sametaylak** [comment]
That's not helpful at all. I wanna keep searchbox(v-text-field) in the top while scrolling rest of the list items.

**Comment by KaelWD** [comment]
position:sticky?
